tools/LoadPackage.groovy

```diff
@@ -18,7 +18,7 @@ class LoadPackage {
     private final String PATH_TO_SCHEDULER_CREDENTIALS_FILE = "config/authentication/admin_user.cred"
     private final String LOAD_PACKAGE_SCRIPT_NAME = "LoadPackage.groovy"
     private String BUCKET_OWNER = "GROUP:public-objects"
-    private final String TMP_DIR_PREFIX = "package_temp_dir"
+    private final String TMP_DIR_PREFIX = "packageTempDir"
```

Comment:
why changing the name? what does it bring?

Reply:
it brings that, we had before a mix between xxx_yyy and xxxYYY